scalastyle-config.xml

```diff
@@ -19,7 +19,7 @@
  <check level="error" class="org.scalastyle.scalariform.SpacesBeforePlusChecker" enabled="true"></check>
  <check level="error" class="org.scalastyle.file.FileLineLengthChecker" enabled="true">
   <parameters>
-   <parameter name="maxLineLength"><![CDATA[110]]></parameter>
+   <parameter name="maxLineLength"><![CDATA[180]]></parameter>

```

Comment:
Okay, I disagree with this one.  I hate hate long runaway lines!   Code is far more readable when lines are shorter.  Though I'm sure you know this and there's some reason why this is being increased.